### PR TITLE
Mysql provider

### PR DESCRIPTION
**Pull request by mlimongelli**
Implmented provider for mySql

### CODE REVIEW

File: src/MySql.Tests/Resources/Init2.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (PascalSenn) [inline src/MySql.Tests/Resources/Init2.sql:1]: empty line

File: src/MySql/SqlCommandBase.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                AttachStdout = true,
+                Cmd = GetCommand(query, settings),
+                Detach = false,
+                Tty = false
+            };
+        }
+
+        private IList<string> GetCommand(
+                string query,
+                ContainerResourceSettings settings)
```
Comment (PascalSenn) [inline src/MySql/SqlCommandBase.cs:27]: format

File: src/MySql/CreateDbCommand.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        private CreateDbCommand(
+            string dbname,
+            ContainerResourceSettings settings)
+        {
+            Parameters = GetContainerExecParameters(
+                $"CREATE DATABASE {dbname};" +
+                $"CREATE ROLE developer_{dbname};" +
+                $"GRANT alter,create,delete,drop,index,insert,select,update,trigger," +
+                $"alter routine,create routine, execute, create temporary tables " +
+                $"ON {dbname}.* TO '{settings.Username}';",
```
Comment (PascalSenn) [inline src/MySql/CreateDbCommand.cs:22]:
```
                $@"CREATE DATABASE {dbname};
                   CREATE ROLE developer_{dbname};
                   GRANT alter,create,delete,drop,index,insert,select,update,trigger,alter routine,
                            create routine, execute, create temporary tables 
                   ON {dbname}.* 
                   TO '{settings.Username}';",
```

**Review by PascalSenn — CHANGES_REQUESTED** [review]

**Review by joslat — APPROVED** [review]
like how simple it is.

File: src/MySql/CreateDbCommand.cs (hunk 3 above)
Comment (Carael) [reply src/MySql/CreateDbCommand.cs:22]: Yeah that's much better.

**Review by PascalSenn — APPROVED** [review]

**Pull request merged**